crypt-static.c

```diff
@@ -35,7 +43,30 @@ SYMVER_crypt;
 #endif
 
 #if INCLUDE_fcrypt
+#if ENABLE_OBSOLETE_API_ENOSYS
+#if ENABLE_FAILURE_TOKENS
+char *
+fcrypt (ARG_UNUSED (const char *key), const char *setting)
+#else
+char *
+fcrypt (ARG_UNUSED (const char *key), ARG_UNUSED (const char *setting))
+#endif
+{
+  /* Pretend this function is not supported.  */
+  errno = ENOSYS;
+
+#if ENABLE_FAILURE_TOKENS
+  /* Return static buffer filled with a failure-token.  */
+  static char retval[3];
+  make_failure_token (setting, retval, 3);
+  return retval;
+#else
+  return NULL;
+#endif
+}
+#else
```

Comment:
Same concern here as for `setkey_r`.

Reply:
Why?  We only define `fcrypt` as function, when we build it as a stub.  Otherwise it is a `strong_alias` for `crypt`.

Reply:
I can just find one function head for `fcrypt` here…  :/

Reply:
As of 0919e33d0e12bf580ce389456e73ce3b88afd771 it seems to be fine.